ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -94,24 +140,74 @@ tests:
   container:
     from: src
   run_if_changed: ^(rpm/*|packaging/.*|Makefile)
-- as: e2e-openshift-conformance-reduced
+- as: rebase-on-nightlies
+  cron: 0 5 * * 1-5
+  steps:
+    workflow: openshift-microshift-rebase
+- as: e2e-openshift-conformance-reduced-rhel8-x86-64
+  cluster: build02
+  run_if_changed: ^(assets/.*|cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile|.*\.sh)
+  steps:
+    cluster_profile: gcp
+    env:
+      GOOGLE_COMPUTE_IMAGE_NAME: rhel-8-v20230202
+      GOOGLE_COMPUTE_MACHINE_TYPE: e2-standard-8
+    workflow: openshift-microshift-e2e-openshift-conformance-reduced
+- as: e2e-reboot-rhel8-x86-64
+  cluster: build02
+  run_if_changed: ^(assets/.*|cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
+  steps:
+    cluster_profile: gcp
+    env:
+      GOOGLE_COMPUTE_IMAGE_NAME: rhel-8-v20230202
+      GOOGLE_COMPUTE_MACHINE_TYPE: e2-standard-8
+    workflow: openshift-microshift-e2e-reboot-tests
+- as: e2e-router-smoke-test-rhel8-x86-64
+  cluster: build02
+  optional: true
+  run_if_changed: ^(assets/.*|cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile|.*\.sh)
+  steps:
+    cluster_profile: gcp
+    env:
+      GOOGLE_COMPUTE_IMAGE_NAME: rhel-8-v20230202
+      GOOGLE_COMPUTE_MACHINE_TYPE: e2-standard-8
+    workflow: openshift-microshift-e2e-router-smoke-test
+- as: e2e-rpm-install-rhel-9-x86-64
```

Comment:
Not sure on the "rhel9" part of the name. A release is tied to an OS version, should we keep it uniform between branches to have the names look the same, changing only the configuration?